Accordion.d.ts

```diff
@@ -53,6 +53,19 @@ export interface AccordionProps<T> {
    */
   onChange(indexes: number[]): void;
 
+  /**
+   *
+   * @param item
+   * @param index
+   */
+  keyExtractor(item: T, index: number): number | string;
+
+  /**
+   * Controls how many columns should be rendered. Multiple columns support
+   * not tested, defaults to 1.
+   */
+  numColumns?: number;
+
```

Comment:
Instead of adding this here, we should extend the interface from FlatListProps from react-native

Reply:
Am definitely with you - but would have to do a deep dive into the [@typed/react-native](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts) as I'm quite new to TS % ).